Makefile

```diff
@@ -44,8 +44,8 @@ build/prettier-el-min.js: prettier-el.js externs.js
 		--externs externs.js \
 		--hide_warnings_for=node_modules/diff-match-patch \
 		--dependency_mode=NONE \
-		--language_out=ECMASCRIPT5 \
-		--output_wrapper='%output%;m'
+		--language_out=ECMASCRIPT_2017 \
```

Comment:
Thank you. I'll change a line in README later..
```
It is recommended to provide a recent version of Node since older versions tend to be significantly slower or don't work (minimum requirement is >=7.x).
```

Reply:
Note that Prettier 2.0 is for Node >=10.13.0 and Prettier 3 is for Node >=14. 
https://prettier.io/blog/2020/03/21/2.0.0.html#drop-support-for-node-versions-older-than-10-6908httpsgithubcomprettierprettierpull6908-by-fiskerhttpsgithubcomfisker
https://prettier.io/blog/2023/07/05/3.0.0#drop-support-for-nodejs-10-and-12-11830httpsgithubcomprettierprettierpull11830-by-fiskerhttpsgithubcomfisker-13118httpsgithubcomprettierprettierpull13118-by-sosukesuzukihttpsgithubcomsosukesuzuki